src/idvc/dvc_interface.py

```diff
@@ -1243,7 +1243,7 @@ def createRegistrationViewer(self):
         self.orientation = self.vis_widget_2D.frame.viewer.getSliceOrientation()
```

Comment:
although the results are the same, the plot is shifted in opposite way. I have checked this when I implemented the automatic registration to be consistent with the signs given by the manual registration. Now, I do not know if the convention was wrong before.

new registration

![new_PR](https://github.com/TomographicImaging/iDVC/assets/138598662/781ccda5-78a8-4a7b-a5e4-61a5049c54c3)

old registration
![old master](https://github.com/TomographicImaging/iDVC/assets/138598662/d9bb23a5-d13e-4653-a497-440ddfe81244)

Reply:
I opened an issue about this and we will address it later 
https://github.com/TomographicImaging/iDVC/issues/298